improver_tests/nbhood/circular_kernel/test_CircularNeighbourhood.py

```diff
@@ -200,32 +178,18 @@ def test_single_point_range_5(self):
         expected = np.ones_like(self.cube.data)
         for index, slice_ in enumerate(SINGLE_POINT_RANGE_5_CENTROID):
             expected[3 + index][3:12] = slice_
-        ranges = 5
-        result = CircularNeighbourhood(weighted_mode=True).apply_circular_kernel(
-            self.cube, ranges
-        )
+        radius = 10000
+        result = Neighbourhood("circular", weighted_mode=True).run(self.cube, radius)
         self.assertArrayAlmostEqual(result.data, expected)
 
     def test_single_point_range_5_small_domain(self):
-        """Test behaviour - non-zero point, small domain, large range.
-        This exhibits the undesirable edge reflection behaviour."""
-
+        """Test code fails if radius larger than the domain."""
```

Comment:
The circular kernel is now constrained to only accept radii smaller than the domain, in the same way as the square kernel